src/util/rmalloc.h

```diff
@@ -23,15 +23,6 @@ static inline char *rm_strdup(const char *s) {
   return RedisModule_Strdup(s);
 }
 
-static char *rm_strndup(const char *s, size_t n) {
```

Comment:
We're not using this function?

Reply:
No, we use a fair amount of `strdup` (and have switched over to `rm_strdup` in some areas), but we don't have any calls to `strndup`, the variant that copies up to N characters.